src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -230,12 +233,30 @@ const mapDispatchToProps = {
 
 FormPage.propTypes = {
   form: PropTypes.object.isRequired,
+  appStateData: PropTypes.shape({}),
+  blockScrollOnMount: PropTypes.bool,
+  contentAfterButtons: PropTypes.element,
+  contentBeforeButtons: PropTypes.element,
+  formContext: PropTypes.shape({
```

Comment:
Do you need formContext here still?

Reply:
It's still being used by the `CustomPage.onReviewPage` prop.